examples/tests/valid/api.w

```diff
@@ -0,0 +1,12 @@
+bring cloud;
+
+let api = new cloud.Api();
+
+let handler = inflight (message: str): str => {
+  return "Hello World";
+};
+// failed when using '/' at the beginning of the path
```

Comment:
> `// failed when using '/' at the beginning of the path`

Not sure I understand this comment

Reply:
this is leftover from the old implementation where I had to create multiple `apigateway_resources` which mustn't contain the slash. Now with the openapi approach this is explicitly needed (which makes the implementation easier)